test/uri/uri_path_test.cc

```diff
@@ -9,25 +9,25 @@ TEST(URI_path, https_example_url_no_path) {
 TEST(URI_path, https_example_url_slash) {
   const sourcemeta::jsontoolkit::URI uri{"https://example.com/"};
   EXPECT_TRUE(uri.path().has_value());
-  EXPECT_EQ(uri.path().value(), "/");
+  EXPECT_EQ(uri.path().value(), "");
 }
 
 TEST(URI_path, https_example_url_single) {
   const sourcemeta::jsontoolkit::URI uri{"https://example.com/foo"};
   EXPECT_TRUE(uri.path().has_value());
-  EXPECT_EQ(uri.path().value(), "/foo");
+  EXPECT_EQ(uri.path().value(), "foo");
 }
 
 TEST(URI_path, https_example_url_multi) {
   const sourcemeta::jsontoolkit::URI uri{"https://example.com/foo/bar/baz"};
   EXPECT_TRUE(uri.path().has_value());
-  EXPECT_EQ(uri.path().value(), "/foo/bar/baz");
+  EXPECT_EQ(uri.path().value(), "foo/bar/baz");
 }
 
 TEST(URI_path, relative_multi) {
   const sourcemeta::jsontoolkit::URI uri{"../foo/bar"};
   EXPECT_TRUE(uri.path().has_value());
-  EXPECT_EQ(uri.path().value(), "/../foo/bar");
+  EXPECT_EQ(uri.path().value(), "../foo/bar");
```

Comment:
Hmmm let me check that

Reply:
What I don't really understand with this is that we have this:

```cpp
TEST(URI_path, without_scheme) {
  const sourcemeta::jsontoolkit::URI uri{"example.com/foo"};
  EXPECT_EQ(uri.path().value(), "example.com/foo");
}
``` 

and we would to have this on the other side:

```
TEST(URI_path, relative_multi) {
  const sourcemeta::jsontoolkit::URI uri{"../foo/bar"};
  EXPECT_TRUE(uri.path().has_value());
  EXPECT_EQ(uri.path().value(), "/../foo/bar");
}
```

I identifier both of the cases as:
- NON URN
- NON tag
- WHICH have not scheme.

So I think we should have a take here.

CC @jviotti

Reply:
Yeah, good point. I think how you have them now (with the current changes to these tests) looks good to me.